main.js

```diff
@@ -212,7 +211,12 @@ function createDefaultMenu() {
           type: 'separator'
         },
         {
-          role: 'quit'
+          label: 'Quit',
+          accelerator: 'CmdOrCtrl+Q',
+          click: function () {
+            quitOnClose = true;
```

Comment:
Rather than using this quitOnClose variable, could we just exit with a nonzero exit code to signify closing?
```js
app.exit(1);
```
and then check for that exit code in the close event listener?